packages/serverless/src/awslambda.ts

```diff
@@ -21,14 +37,26 @@ export type AsyncHandler<T extends Handler> = (
   context: Parameters<T>[1],
 ) => Promise<NonNullable<Parameters<Parameters<T>[2]>[1]>>;
 
-interface WrapperOptions {
+export interface WrapperOptions {
   flushTimeout: number;
   rethrowAfterCapture: boolean;
   callbackWaitsForEmptyEventLoop: boolean;
   captureTimeoutWarning: boolean;
   timeoutWarningLimit: number;
 }
 
+export const defaultIntegrations: Integration[] = [...sdk.defaultIntegrations, new AWSServices()];
```

Comment:
Can we use `Sentry.` namespace instead of `sdk`? We use this naming convention in all other places. Not necessary, but it'd be good to have it unified.